app/assets/javascripts/test/puppeteer/dataset_rendering_helpers.js

```diff
@@ -56,7 +56,7 @@ async function waitForRenderingFinish(page: Page) {
   await removeFpsMeter(page);
 
   let currentShot;
-  let lastShot = await page.screenshot();
+  let lastShot = await page.screenshot({ fullPage: true });
```

Comment:
Why did this become necessary?

Reply:
I already added this in line 64 when fixing the screenshot tests the first time and forgot to add it here.
This became necessary because we're taking a screenshot of the page now and without setting fullPage to true I didn't find out the dimensions of this screenshot (which I need when calling pixelmatch). Chrome's browser window seems to be smaller than the viewport.

Reply:
Ok, cool :)